### PR TITLE
Add Nube colors and Nube logo

### PR DESCRIPTION
**Pull request by nguyenoanh5597**

### CODE REVIEW

File: src/components/HomepageFeatures/index.tsx
Comment (Quynh-Nguyen) [inline src/components/HomepageFeatures/index.tsx]: @nguyenoanh5597 
Hi,

I think we should keep the coding convention is single-quote

**Review by Quynh-Nguyen — CHANGES_REQUESTED** [review]

File: src/components/HomepageFeatures/index.tsx
Comment (nguyenoanh5597) [reply src/components/HomepageFeatures/index.tsx]: fixed

**Comment by Quynh-Nguyen** [comment]
@nguyenoanh5597 
I don't see the logo update on this PR, so this is all updated and ready to merge, right?

**Comment by oanhnguyen-kms** [comment]
> @nguyenoanh5597 I don't see the logo update on this PR, so this is all updated and ready to merge, right?

@Quynh-Nguyen just saw the logo problem, pr should only merge once the logo has been updated, so don't merge now

**Comment by nguyenoanh5597** [comment]
hi @Quynh-Nguyen, please help to review again

image

**Review by Quynh-Nguyen — APPROVED** [review]
It's okay for now.
A little feedback, the logo relatively not fit with the menu.
If you can modify it to bring it up to around 10 pixels, and make the logo will be on the same line as the menu, it will be great!

**Pull request merged**
